<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MacBook-Pro-6.local" ignored="0" name="简单测试2" tests="2" failures="1" timestamp="25 二月 2021 06:39:14 GMT" time="10.014" errors="0">
  <testcase name="testCase1" time="5.012" classname="com.test.api.test3">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;2&amp;gt; but was:&amp;lt;3&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<2> but was:<3>
at com.test.api.test3.testCase1(test3.java:37)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testCase1 -->
  <testcase name="testCase2" time="10.002" classname="com.test.api.test3"/>
</testsuite> <!-- 简单测试2 -->
